user_docs/en/userGuide.md

```diff
@@ -2163,41 +2163,17 @@ This means that you do not have to scroll the display at the end of each line ev
 This may allow for more fluent reading of large amounts of text.
 It is disabled by default.
 
-##### Show paragraph start {#BrailleShowParagraphStart}
+##### Paragraph start markers {#BrailleParagraphStartMarkers}
+
 | . {.hideHeaderRow} |.|
 |---|---|
-|Options |Default (Disabled), Enabled, Disabled|
-|Default |Disabled|
+|Options |Don't use paragraph start markers, Double space, Pillcrow|
+|Default |Don't use paragraph start markers|
 
-If enabled and "Read by paragraph" is checked, two spaces will be displayed to indicate the start of a paragraph.
+If "Read by paragraph" is checked, the selected start marker will be displayed to indicate the start of a paragraph.
 This can be especially helpful in applications used to read large pieces of text, like structured documents or books.
 In such documents, knowing where paragraphs start maybe important to understand the structure of the content, or to set bookmarks or annotations based on paragraph position.
```

Comment:
>or to set bookmarks or annotations based on paragraph position.

Is this through the text editor UX, or are you referring to add-on functionality?

Reply:
This is related to applications to read books, for example,Thorium Reader. We can press NVDA+enter to perform a click (activate the current object), sothat the app knows the current element of the DOM (for the current HTML document of the epub being read), and then press control+b toset a bookmark. Then pressing control+n we can open the navigation menu, use the arrow keys to select the bookmarks tab, and create an annotation pressing the edit button, adding several lines of text. If we do this in the middle of a paragraph, the bookmark isadded based on the HTML element, so it won't be very precise. The placeMarkers add-on (maintained by me) considers the number of characters from the start of the text, but bookmarks are added for each document with a different title, so in Thorium I prefer to use the reading app feature, since chapters of books areplaced in separate HTML documents, but the title is for the whole book.